sys/cloudformation/parameters.prod.json

```diff
@@ -1,13 +1,22 @@
 [
   {"ParameterKey": "ELBCertificateArn", "ParameterValue": ""},
-  {"ParameterKey": "EcrImageTagNginx", "ParameterValue": "1603920252"},
-  {"ParameterKey": "EcrImageTagPhp", "ParameterValue": "1603916520"},
-  {"ParameterKey": "EcrImageTagPhpCanary", "ParameterValue": "1603916520"},
+  {"ParameterKey": "EcrImageTagNginx", "ParameterValue": "1605006094"},
+  {"ParameterKey": "EcrImageTagPhp", "ParameterValue": "1605006094"},
+  {"ParameterKey": "EcrImageTagPhpCanary", "ParameterValue": "1605006094"},
   {"ParameterKey": "Environment", "ParameterValue": "prod"},
   {"ParameterKey": "ExecRoleArn", "ParameterValue": ""},
   {"ParameterKey": "Subnets", "ParameterValue": ""},
   {"ParameterKey": "VpcId", "ParameterValue": ""},
 
   {"ParameterKey": "EnvAPPENV", "ParameterValue": "prod"},
-  {"ParameterKey": "EnvREDISURL", "ParameterValue": ""}
+  {"ParameterKey": "EnvAPPENVCanary", "ParameterValue": "canary"},
+  {"ParameterKey": "EnvREDISURL", "ParameterValue": ""},
+  {"ParameterKey": "EnvREDISHOST", "ParameterValue": ""},
+  {"ParameterKey": "EnvAPPSECRET", "ParameterValue": ""},
+  {"ParameterKey": "EnvBITBUCKETSECRET", "ParameterValue": ""},
+  {"ParameterKey": "EnvBITBUCKETTOKEN", "ParameterValue": ""},
+  {"ParameterKey": "EnvCIRCLECITOKEN", "ParameterValue": ""},
+  {"ParameterKey": "EnvGITHUBUSERNAME", "ParameterValue": ""},
```

Comment:
Parameters having defaults are intended for avoiding params pollutions .. why to add a default param here if the value passed is the default one?

Reply:
it is "useful" as I have a (temporary) copy with the secrets:

```json
[
  {"ParameterKey": "ELBCertificateArn", "ParameterValue": "arn:aws:acm:***:***:certificate/***"},
  {"ParameterKey": "EcrImageTagNginx", "ParameterValue": "1605220368"},
  {"ParameterKey": "EcrImageTagPhp", "ParameterValue": "1605220368"},
  {"ParameterKey": "EcrImageTagPhpCanary", "ParameterValue": "1605220368"},
  {"ParameterKey": "Environment", "ParameterValue": "prod"},
  {"ParameterKey": "ExecRoleArn", "ParameterValue": "arn:aws:iam::***:role/ecsTaskExecutionRole"},
  {"ParameterKey": "Subnets", "ParameterValue": "subnet-***,subnet-***,subnet-***"},
  {"ParameterKey": "VpcId", "ParameterValue": "vpc-***"},

  {"ParameterKey": "EnvAPPENV", "ParameterValue": "prod"},
  {"ParameterKey": "EnvAPPENVCanary", "ParameterValue": "canary"},
  {"ParameterKey": "EnvREDISURL", "ParameterValue": "redis://***.cache.amazonaws.com:6379"},
  {"ParameterKey": "EnvREDISHOST", "ParameterValue": "***.cache.amazonaws.com"},
  {"ParameterKey": "EnvAPPSECRET", "ParameterValue": "***"},
  {"ParameterKey": "EnvBITBUCKETSECRET", "ParameterValue": "***"},
  {"ParameterKey": "EnvBITBUCKETTOKEN", "ParameterValue": "***"},
  {"ParameterKey": "EnvCIRCLECITOKEN", "ParameterValue": "***"},
  {"ParameterKey": "EnvGITHUBUSERNAME", "ParameterValue": "***"},
  {"ParameterKey": "EnvSENTRYDSN", "ParameterValue": "https://***@***.ingest.sentry.io/***"},
  {"ParameterKey": "EnvDARKCANARYTHRESHOLD", "ParameterValue": "25"}
]
```

Reply:
I don't get the point .. you can have your copy with 100 params, and an "official" one with just 2 ...